### PR TITLE
Proxy API For Mkube

### PR DESCRIPTION
**Pull request by dvaldivia**
Proxies the requests for `/api/v1/clusters` to an mkube endpoint

### CODE REVIEW

File: restapi/mkube.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+	apiErrors "github.com/go-openapi/errors"
+)
+
+// serverMkube handles calls for mkube
+func serverMkube(w http.ResponseWriter, req *http.Request) {
+	// destination of the request, the mkube server
+	targetURL := fmt.Sprintf("%s%s", getM3Host(), req.URL.String())
+
+	client := &http.Client{}
```
Comment (Alevsk) [inline restapi/mkube.go:34]: this can be in another PR, but client should be able to configure TLS communication via http.Transport in case m3 requires tls authentication, ca certificate, etc.

File: restapi/mkube.go [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+
+	apiErrors "github.com/go-openapi/errors"
+)
+
+// serverMkube handles calls for mkube
+func serverMkube(w http.ResponseWriter, req *http.Request) {
```
Comment (cesnietor) [inline restapi/mkube.go:30]: please add tests for this

Comment (dvaldivia) [reply restapi/mkube.go]: done

File: restapi/mkube_test.go [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+package restapi
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+func Test_serverMkube(t *testing.T) {
+
```
Comment (cesnietor) [inline restapi/mkube_test.go:26]: which function are you testing?

Comment (dvaldivia) [reply restapi/mkube_test.go:45]: I messed that up, but it's fixed now

Screen Shot 2020-05-27 at 12 31 30 PM
